20151014/python/code_RW.py

```diff
@@ -0,0 +1,61 @@
+#%pylab inline
+import numpy as np
+import os.path
+import glob
+import matplotlib.pyplot as plt
+
+
+############################################################################################################################
+
+# Function definitions
+
+def read_files():
```

Comment:
Maybe include variables for the function, so that the function can be called with different paths, or folders for example.
ie. def read_files(path, folder_name): ....
And then you can use:
my_path = os.path.join(.....)
my_folder = '2015-05-12'
header1, data1 = read_files(my_path, my_folder)